### PR TITLE
[Tests] Link hooks to the index script

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's link hooks to the index script.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: src/Tests/index.ts [hunk 1]
```diff
@@ -80,9 +81,8 @@ export function run(): Promise<void> {
 
   const testsRoot = path.resolve(__dirname, '.');
 
-  // adds hooks first
-  const hooks = 'hooks.js';
-  mocha.addFile(path.join(testsRoot, hooks));
+  mocha.globalSetup(mochaHooks.suiteSetup);
```
[the remark this answers is not in the document]
Comment (YongseopKim) [reply src/Tests/index.ts:84]: That sounds reasonable! I'll close this first.

**Comment by yunjayh** [comment]
> @yunjayh Q. how do tests rerun?

I removed the label, and then force pushed.

**Comment by YongseopKim** [comment]
@yunjayh is it impossible by a way like calling bot? like `nncc test`

**Comment by yunjayh** [comment]
No it not implemented. :sob: Is `nncc test` triggered by jenkins?

**Comment by YongseopKim** [comment]
Anyway I close

**Pull request closed without merging**